tests/resources/transaction/test_tx.py

```diff
@@ -484,6 +493,37 @@ def test_negative_timestamp(self):
         data = response.json_value()
         self.assertFalse(data['success'])
 
+    @inlineCallbacks
+    def test_partially_validated_not_found(self):
+        # Tx tesnet 0033784bc8443ba851fd88d81c6f06774ae529f25c1fa8f026884ad0a0e98011
```

Comment:
You told me this comment was copied from some other test.

Reply:
Removed.